spec/fixtures/officehub.conf

```diff
@@ -0,0 +1,2 @@
+host: 10.64.12.39
```

Comment:
Should we anonymise this with a fake host and key, but place this info somewhere on confluence for internal consumption?

Reply:
Even if we do it would still be in the history. A little late at this point.

Reply:
I dont think this is a problem, its an internal guest network address and key. As this code is used often within the office having something that works out of the box is nice.